src/app/tracing/dialog/dialog-movable/dialog-movable.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, Inject } from "@angular/core";
+import {
+    MAT_LEGACY_DIALOG_DATA as MAT_DIALOG_DATA,
+    MatLegacyDialogRef as MatDialogRef,
+} from "@angular/material/legacy-dialog";
+
+export enum DialogMovableTemplate {
+    fclStationProperties = "fclStationProperties",
+    dataImportWarning = "dataImportWarning",
+}
+
+interface DialogMovableData {
+    data: any;
+}
+
+@Component({
+    selector: "fcl-dialog-movable",
+    templateUrl: "./dialog-movable.component.html",
+})
+export class DialogMovableComponent {
+    dialogData: any;
+    constructor(
```

Comment:
Code is obsolete

Reply:
remove obsolete code

Reply:
@ruegenbfr the entire contructor is obsolete? but don't we have to inject MAT_DIALOG_DATA here?

Reply:
I think I got it, I need to add it to the warning component

Reply:
We don't use the movable dialog component itself as dialog only as a layout component. We refer to it only by its selector. So the removal of the constructor should have no consequences.